src/main/java/network/brightspots/rcv/TieBreak.java

```diff
@@ -51,35 +51,27 @@
 class TieBreak {
 
   private static Random random;
-
-  // list of all candidates who are tied for this tie-breaker
   private final List<String> allTiedCandidates;
-  // mode we're using for breaking ties
   private final Tabulator.TieBreakMode tieBreakMode;
   // ordering to use if we're doing permutation-based tie-breaking
   private final ArrayList<String> candidatePermutation;
-  // round in which this tiebreak occurred
   private final int round;
   // number of votes the tying candidates received
   private final BigDecimal numVotes;
   // roundTallies: map from round number to map of candidate ID to vote total (for that round)
   // e.g. roundTallies[1] contains a map of candidate IDs to tallies for each candidate in round 1
   private final Map<Integer, Map<String, BigDecimal>> roundTallies;
   private boolean selectingAWinner;
-  // candidate ID selected to lose/win the tiebreak
   private String selectedCandidate;
-  // reason for the selection
   private String explanation;
 
-  // function: TieBreak
-  // purpose: TieBreak constructor stores member data
-  // param: selectingAWinner whether we're determining a winner (as opposed to a loser)
+  // TieBreak constructor
+  // param: selectingAWinner are we determining a winner or loser
```

Comment:
just curious how you decide when to list out the params like this vs. removing them

Reply:
Generally I kept param comments when the meaning was not obvious, and removed them when it was.  In this case selectingAWinner being false does not naturally imply selecting a loser so the comment is important.  I confirmed this with Eric.